pytest/tests/sanity/congestion_control.py

```diff
@@ -254,10 +252,7 @@ def __load(self, node: BaseNode, sender_account, target_account):
             with self.lock:
                 self.txs.append((sender_account.account_id, tx_hash))
 
-            # This sleep here is more a formality, the call_contract call is
-            # slower. This is also the reason for sending transactions from
-            # multiple threads.
-            time.sleep(0.1)
+            time.sleep(0.5)
```

Comment:
So the transactions are now sent at a 5x times slower rate? Is this because fetching the blocks makes the request 5x faster?

Reply:
Using sleep for rate limiting feels a bit fragile.
Maybe measuring time between requests would be a bit more robust? Something like "if last_request_start_time.elapsed() > 0.2s {start_new_request()};"

But as long as it works...

Reply:
Previously the transaction sending was the bottle neck so the 0.1 was negligible. 
Currently the transaction sending is very quick so the rate is controlled by the sleeps. 

In my test runs it was very reliable, in each try the number of submitted transactions was the same. I think it's good enough for now so let's keep it as is.